lib/livebook/notebook/learn/intro_to_explorer.livemd

```diff
@@ -0,0 +1,83 @@
+# Data transform with Explorer
+
+```elixir
+Mix.install([
+  {:explorer, "~> 0.5.4"},
+  {:kino_explorer, "~> 0.1.1"}
+])
+```
+
+## Introduction
+
+KinoExplorer makes it intuitive to build data transform pipelines by seamlessly integrating Explorer and Livebook.
+To work with it in Livebook we need two libraries:
+
+* The [`explorer`](https://github.com/elixir-nx/explorer)
+  package brings series (one-dimensional) and dataframes (two-dimensional) for fast data exploration to Elixir.
+
+* The [`kino_explorer`](https://github.com/livebook-dev/kino_explorer) package automatically renders an `Explorer.DataFrame` or `Explorer.Series` as a data table.
+
+The generated code make extensive use of `Explorer.DataFrame`'s functions, so it's handy to alias the module to something shorter and it will respect it.
+
+```elixir
+alias Explorer.DataFrame, as: DF
+```
+
+<!-- livebook:{"branch_parent_index":0} -->
+
+## The Data transform smart cell
```

Comment:
I think we can have two sections:

1. one section shows how to get started with explorer. we can pick the iris dataset and show one or two operations, and then discuss how the table works

2. then have this smart cell section showing how to further manipulate data